fcs/samples/EditorService/EditorService.fsproj

```diff
@@ -13,7 +13,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="4.6.2" />
-    <PackageReference Include="FSharp.Core" Version="4.5.2" />
     <PackageReference Include="FSharp.Core" Version="4.1.19" />
```

Comment:
I suspect FSharp.Core 4.1.19 appeared as a merge error and can be cleaned up, too.  Same for other projects.

Reply:
Don submitted a pr that addresses this.